internal/learn/about-iota/data-transfer.md

```diff
@@ -141,6 +151,12 @@ All this information must be created by a client (wallet software or other progr
 
 ### Message Validation
 
+:::note
+
+The message validation rules have been updated to reflect the upcoming Stardust release.
+
+:::
+
```

Comment:
Same here. I'd not describe what has been changed. I'd document the actual change and leave the documentation below related to mainnet.